docs/source/release-notes.rst

```diff
@@ -3,7 +3,7 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
-FiftyOne Teams 2.6.1
+FiftyOne Enterprise 2.6.1
```

Comment:
Looks like there's a few more `-----` lengths that need updating to match the new headings. Gotta suppress those doc warnings! 🤓